snmp/datadog_checks/snmp/data/default_profiles/netgear-readynas.yaml

```diff
@@ -101,11 +101,11 @@ metrics:
           name: netgear.readynasos.volumeStatus
         tag: netgear_readynasos_volume_status
         mapping:
-          1: redundant
-          2: unprotected
+          1: online
+          2: offline
           3: degraded
-          4: dead
-          5: inactive
+          4: faulted
+          5: removed
```

Comment:
what's the source of this mapping ?
we have this metric in two different MIBs:
- [READYNASOS-MIB](https://github.com/DataDog/snmp-mibs/blob/main/mibs/READYNASOS-MIB) in which the mapping is 
1: REDUNDANT
				2: UNPROTECTED
                                3: DEGRADED
                                4: DEAD
			        5: INACTIVE
                                6: UNKNOWN
- [READYDATAOS-MIB](https://www.circitor.fr/Mibs/Html/R/READYDATAOS-MIB.php) in which the mapping is 
1: ONLINE
2: OFFLINE 
3: DEGRADED 
4: FAULTED 
5: REMOVED 
6: UNAVAILABLE 
7: UNKNOWN

but in here we are using the READYNASOS-MIB so the mapping should be the initial one

Reply:
That's interesting, I suppose I broke it when rebased on master... Fixed